ssl/quic/quic_obj_local.h

```diff
@@ -324,5 +327,15 @@ ossl_quic_obj_get0_port_leader(const QUIC_OBJ *obj)
         : NULL;
 }
 
+/*
+ * Change the domain flags. Should only be called immediately after
+ * ossl_quic_obj_init().
+ */
+static ossl_inline ossl_unused void
+ossl_quic_obj_set_domain_flags(QUIC_OBJ *obj, uint64_t domain_flags)
+{
+    obj->domain_flags = domain_flags;
```

Comment:
Should this be modified to return an error if the domain flags are attempted to be set after its no longer valid to do so, or if a user attempts to set a combination of conflicting flags?  I don't think its critical, but may be something to consider before we merge this

Reply:
I would say, open a new issue for that.